src/python/turicreate/test/test_image_classifier.py

```diff
@@ -74,7 +76,15 @@ def get_test_data():
         images.append(tc_image)
 
     labels = ["white"] * 5 + ["black"] * 5
-    return tc.SFrame({"awesome_image": images, "awesome_label": labels})
+    data_dict = {"awesome_image": images, "awesome_label": labels}
+    data = tc.SFrame(data_dict)
+
+    for model_name, feature_length in MODEL_TO_FEATURE_SIZE_MAPPING.items():
+        if str(feature_length) in data_dict.keys():
+            continue
+        data[str(feature_length)] = get_deep_features(data["awesome_image"], model_name)
+
+    return data
```

Comment:
So changing `get_test_data` to return deep features probably isn't what we want. We want test cases for creating model with deep features and test cases for creating models with images. If we make this change, then we'll only have test cases for creating models with deep features.

I would probably create new sub-classes that call `get_deep_features`, in their `setUpClass` method, before calling `tc.image_classifier.create`.

Reply:
I agree, that's why I added one more column to the existing sframe data to use this for testing models with both Images and deep features. 
If you look at the columns that are being generated by the function `get_test_data` it has columns `awesome_image`, `awesome_label`, `2048` and `1000`. So that we can use the same data for testing both models with Images and deep features.